Common/Currencies.cs

```diff
@@ -21,24 +21,47 @@ namespace QuantConnect
     /// <summary>
     /// Provides commonly used currency pairs and symbols
     /// </summary>
-    public static class Currencies
-    {
+    public static class Currencies {
```

Comment:
I didn't get this other style issue. Sorry. 
You may want to fix the last commit to include this: `git commit --fixup=65a86b1`

Reply:
Sharp eyes. No problem, .. will have to reconfigure my editor it seems.

Reply:
love seeing the `git fu` grow :D